<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookingsearch</class>
 <widget class="QDialog" name="bookingsearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>691</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter ref</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ref2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="init_location"/>
     </item>
     <item>
      <widget class="QLineEdit" name="final_location"/>
     </item>
     <item>
      <widget class="QLineEdit" name="class_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="deptime"/>
     </item>
     <item>
      <widget class="QLineEdit" name="return_time"/>
     </item>
     <item>
      <widget class="QLineEdit" name="startdate"/>
     </item>
     <item>
      <widget class="QLineEdit" name="enddate"/>
     </item>
     <item>
      <widget class="QLineEdit" name="hotel"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
